_posts/2018-03-19-newest-updates-for-tax-data-2018

```diff
@@ -0,0 +1,31 @@
+title: 'Newest Updates for Tax Data 2018'
+author:
+  display_name: mheagin
+  email: mheagin@utah.gov
+date: 2018-03-19 22:38:18
+categories:
+  - SGID Blog
+tags:
+  - Tax
+  - Tax Entities
+  - Tax Areas
+  - Sales Tax Areas
+  - Transit Special Tax Areas
+---
+
+With April just around the corner, it’s tax time again. If you don’t procrastinate and you've already done your taxes, you might be interested in knowing just who is getting that hard-earned money you make. 
+
+Well, we have data for that.
+
+#Newest Tax Entities and Areas for 2017
+This is the final 2017 list of all incorporated bodies that have the power to levy a local property tax: school districts, cities, special service districts&mdash;even mosquito abatement districts. While these entities don't represent exact legal boundaries, they are used by the Utah State Tax Commission, Property Tax Division, for assessing properties. The entities can overlap, but the areas represent individual polygons of intersecting entities. 
+ 
+#Newest Sales Tax Areas for Quarter 1, 2018
```

Comment:
What's the difference between a "sales tax area" and a "sales tax collection area"?